service/src/test/java/com/commercetools/pspadapter/payone/transaction/TransactionBaseExecutorTest.java

```diff
@@ -47,16 +46,15 @@ public void exceptionToResponseJsonString() throws Exception {
         final String TEST_ERROR_MESSAGE = "TEST MESSAGE WOOT";
         String jsonString = TransactionBaseExecutor.exceptionToResponseJsonString(new Exception(TEST_ERROR_MESSAGE));
 
-        assertThat(jsonString, is(not(isEmptyOrNullString())));
+        assertThat(jsonString).isNotEmpty();
 
         JsonNode node = new ObjectMapper().readTree(jsonString);
 
-        assertThat(node.has(STATUS), is(true));
-        assertThat(node.has(ERROR_CODE), is(true));
-        assertThat(node.has(CUSTOMER_MESSAGE), is(true));
+        assertThat(node.has(STATUS)).isTrue();
+        assertThat(node.has(ERROR_CODE)).isTrue();
+        assertThat(node.has(CUSTOMER_MESSAGE)).isTrue();
 
-        assertThat("Error message node should contain exception message",
-                node.get(ERROR_MESSAGE).asText(), containsString(TEST_ERROR_MESSAGE));
+        assertThat(node.get(ERROR_MESSAGE).asText()).as("Error message node should contain exception message").contains(TEST_ERROR_MESSAGE);
```

Comment:
we don't have hard line length limit but we strongly encourage to split long lines

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/184/commits/111a893c304719b668c15c0d09a2c63a8a891abe#diff-d7a6d71c83bdd9a5753a0e9241a935a4R57